op-mode-definitions/show-bridge.xml.in

```diff
@@ -42,6 +42,18 @@
             </properties>
             <command>${vyos_op_scripts_dir}/bridge.py show_fdb --interface=$3</command>
           </leafNode>
+          <leafNode name="details">
+            <properties>
+              <help>Display bridge interface details</help>
+            </properties>
+            <command>vtysh -c "show int $3"</command>
```

Comment:
Could you extend `bridge.py` and move this command as a function, for example, `show_bridge_detail` or `show_detail`
For formatted output `show interface brX` for raw output `show interface br0 json`
As we have new format for the op-mode 
https://phabricator.vyos.net/T4564
https://phabricator.vyos.net/T2719

It is a good idea to use it anywhere where it is possible

Reply:
However I have a problem, I don't use the OS command to get it, in this case I find it more interesting to use the vtysh/frr admin commands to get it without having to port complex processing code.

How do I need to portable py code for this situation?

Reply:
It won't be hard, extend `bridge.py`
A quick example:
```
def _get_bridge_details(iface, raw=False):
    """Get interfaace detail statistics"""
    opt = 'json' if raw else ''
    return call(f'vtysh -c "show interface {iface} {opt}"')

show_bridge_detail(raw: bool, interface: str):
    if raw:
        _get_bridge_details(interface, raw=True)
    else:
        _get_bridge_details(interface)
```

Reply:
done